mysite/settings.py

```diff
@@ -0,0 +1,131 @@
+"""
+Django settings for mysite project.
+
+Generated by 'django-admin startproject' using Django 3.2.18.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.2/ref/settings/
```

Comment:
Same here — add context, remove excessive bits.

Reply:
Addressed https://github.com/kpi-web-guild/django-girls-blog-Martolivna/pull/19/commits/c32a591aa8a9c97adb244fe4adf7087e816c8643 .

Reply:
> It adds a collection of settings for an instance of Django

What's "**It**" in this context?

Reply:
It is this commit.

Reply:
It's unclear from that message. If you want to refer to the current commit, you can type in just that  — "This commit".

Reply:
Avoid implicit references to unclear entities as somebody else wouldn't have the context that it refers to the commit, as it may refer to something else in their mind.

Reply:
Marking this as resolved.